src/index.html

```diff
@@ -10,8 +10,175 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      name="info-form"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input
+          type="text"
+          name="lname"
+          >
```

Comment:
closing brace should be at the same level as opening one if you move attributes to a new line
```suggestion
        <label class="form-field">
          Surname:
          <input
          type="text"
          name="lname"
        >
```